apps/console/src/features/applications/components/wizard/passive-sts-protocol-settings-wizard-form.tsx

```diff
@@ -82,8 +83,8 @@ export const PassiveStsProtocolSettingsWizardForm: FunctionComponent<PassiveStsS
                     submitState={ triggerSubmit }
                 >
                     <Grid>
-                        <Grid.Row columns={ 1 }>
-                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 8 }>
+                        <Grid.Row>
+                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
```

Comment:
Since values for all the breakpoint are same, we can keep only `mobile={ 16 }`

Reply:
Addressed with 37333d9